utils.sh

```diff
@@ -30,8 +30,8 @@ function create_repo() {
 
 function set_policy() {
   local region=$1
-  RepositoryPermissionPolicy=$(printenv | grep RepositoryPermissionPolicy | sed 's/.*=//')
-  if [ -n "$RepositoryPermissionPolicy" ]
+  RepositoryPermissionPolicy=$(printenv RepositoryPermissionPolicy)
```

Comment:
> If RepositoryPermissionPolicy is set in the env, then why would set -u be a problem?

Had to chew on this for probably longer than I should've, as it seemed like a totally valid point. And it was. Issue was actually with my test script where I was **[unsetting](https://github.com/mapbox/ecs-conex/pull/100/commits/39856462fe39f2f3f77e2320e5560bcfd51d52ea)** this variable to test the "no-policy" case. When I switched to testing a realistic no-policy case (`RepositoryPermissionPolicy=""`), just having the `-n` test worked fine!

Sorry it took so long to get to this. 🙃